renpy/display/dragdrop.py

```diff
@@ -171,19 +171,83 @@ class Drag(renpy.display.core.Displayable, renpy.revertable.RevertableObject):
         to the corner of this drag.
 
     `drag_offscreen`
-        If true, this draggable can be moved offscreen. This can be
```

Comment:
The warning should probably stay in.
For example
```
Controls if and how this draggable can be moved offscreen.
This can be dangerous (...)

This should be one of:
(...)
```